man/dockerd.8.md

```diff
@@ -374,12 +374,11 @@ unix://[/path/to/socket] to use.
   Path to the userland proxy binary.
 
 **--userns-remap**=*default*|*uid:gid*|*user:group*|*user*|*uid*
-  Enable user namespaces for containers on the daemon. Specifying "default"
-  will cause a new user and group to be created to handle UID and GID range
-  remapping for the user namespace mappings used for contained processes.
-  Specifying a user (or uid) and optionally a group (or gid) will cause the
```

Comment:
This leaves the `default` option undocumented. Since `default` seems to be the standard case, and the one documented in the RH guides I've seen, how about something like:

  Specifying "default" will cause a new `dockremap` user and group to be created (if necessary) but will not actually create the necessary entries in the subuid(5) and subgid(5) files.

Bleagh. No. Yuk. I'm sorry, I need to run out the door now, but I'll check back in tomorrow morning. Maybe sleep will bring new insights. I'm sorry this has turned out so complicated.